packages/example_hardhat/README.md

```diff
@@ -0,0 +1,79 @@
+# <PACKAGE>
+
+*Include a high level description of this package here*
+
+## Running the project locally
+
+Install dependencies with `yarn`
+
+Testing
+Use `yarn test` inside `packages/<package>` to run tests locally for just this package
+
+Coverage
+Run `yarn coverage` 
+
+Formatting
+TODO
+
+Linting
+TODO
+
+## Package structure and minimum standards
+
+A NOTE ON DEPENDENCIES
+1. Add whatever dependencies you like inside your package; this template is for hardhat usage
+2. For most Pull Requests there should be minimum changes to `yarn.lock` at root level
+3. Changes to root-level dependencies are permissible, however they should not be downgraded
+4. Take care to run `yarn` before pushing your changes
+
+UNIT TESTING
+1. Unit tests are to be added in `packages/<package>/test`
+2. Coverage must meet minimum requirements for CI to pass
+3. Tests must use appropriate namedAccount(s) and contract ROLE set up, at `packages/<package>/hardhat.config.ts`
+4. UnnamedAccounts must be set up for users within unit tests; under no circumstances should tests be written as `deployer`
+5. It's permissible to create mock contracts at `packages/<package>/contracts/mock` e.g. for third-party contracts
+6. Tests must not rely on any deploy scripts from the deploy package; your contracts must be deployed inside the test fixture. See example below.
+
+
+GOOD EXAMPLE
+    ```
+    export const setupLand = withSnapshot([], async function (hre) {
```

Comment:
do we still need wthSnapshot if we deploy manually in the tests ?

Reply:
Nope, I'll update this (per @adjisb example test fixture) to:
```
const Asset = await ethers.getContractFactory('Asset');
const asset = await Asset.deploy(...);
```